test/mystery-encounter/encounter-test-utils.ts

```diff
@@ -193,7 +152,10 @@ async function handleSecondaryOptionSelect(game: GameManager, pokemonNo: number,
  * @param game
  * @param runRewardsPhase
  */
-export async function skipBattleRunMysteryEncounterRewardsPhase(game: GameManager, runRewardsPhase = true) {
+export async function skipBattleRunMysteryEncounterRewardsPhase(
+  game: GameManager,
+  runRewardsPhase?: false | undefined,
+) {
```

Comment:
```suggestion
export async function skipBattleRunMysteryEncounterRewardsPhase(game: GameManager, runRewardsPhase?: false) {
```
Making it optional should automatically add `undefined` to the type signature, and there's no reason to ever manually pass `undefined` to it.